modules/ROOT/pages/install-self-managed.adoc

```diff
@@ -76,14 +76,16 @@ During installation, the `rtfctl` utility displays any errors encountered.
 . The `rtfctl` utility will display the After the installation has completed succesfully, update the Mule license:
 +
 ----
-rtfctl --update mule-license
+rtfctl apply mule-license '<mule-license-key>'
```

Comment:
I think there's more to this step than we document here. If this works like prior versions of rtfctl, we need the user to base64 encode their `license.lic` file and paste that string as the "mule-license-key".
If true, we can copy the procedure from our current docs and place in-line: https://docs.mulesoft.com/runtime-fabric/1.7/replace-license-key#procedure

@sarjeet2013 to confirm.

Reply:
That's correct @sundippatel. The step is same as prior version of rtfctl.

Reply:
Good point. I should have known that. I remember this from version 1.0. ;-) I added the procedures, but as another section so that it's easier to follow.